docs/devices-api/more-info/models.mdx

```diff
@@ -404,6 +415,16 @@ export interface Respiration {
 export interface Sample {
   time: string;
   value: number;
+  data_source?: SourceInfo;
+}
+```
+
+### Source Type
+
+```typescript
+export enum SourceType {
+  manual = "MANUAL_ENTRY",
+  device = "DEVICE",
 }
```

Comment:
now what about updating the sample data with an example?

hint hint https://docs.metriport.com/devices-api/api-reference/get-body-data